pvlib/spa_c_files/README.md

```diff
@@ -1,34 +1,80 @@
 README
 ------
 
-NREL provides a C implementation of the solar position algorithm 
-described in 
-[Reda, I.; Andreas, A. (2003). Solar Position Algorithm for Solar Radiation Applications. 55 pp.; NREL Report No. TP-560-34302](
-http://www.nrel.gov/docs/fy08osti/34302.pdf).
+NREL provides a C implementation of the solar position algorithm described in
+[Reda, I.; Andreas, A. (2003). Solar Position Algorithm for Solar Radiation Applications. 55 pp.; NREL Report No. TP-560-34302](http://www.nrel.gov/docs/fy08osti/34302.pdf).
 
-This folder contains the files required to make NREL's C code accessible
-to the ``pvlib-python`` package. We use the Cython package to wrap NREL's SPA 
+This folder contains the files required to make SPA C code accessible
+to the `pvlib-python` package. We use the Cython package to wrap the NREL SPA 
 implementation. 
```

Comment:
Copy the comment from `solarposition.py` that explains that we replace `timezone` with `time_zone` at build

Reply:
There is already a comment in this readme on line 17-18:
>The SPA C files will be patched to avoid
[Python bug 24643](https://bugs.python.org/issue24643) when the extension is
built.

Should I add more detail than what's already in `setup.py`?

Reply:
It would help, I think.  What a user (or code maintainer sees) is the change in keyword, which isn't easily linked to the Python bug unless someone already knows what bug 24643 is.  Just thinking down the road to when the bug is fixed and we can revert this patch, or when someone asks why we are editing spa.c.

Reply:
ok